File: 1_Período/Lista 13/C/L13_Q01.c
#include <stdio.h>

int main(){
    int matriz[3][3];
    int i, j, soma = 0;

    for(i = 0; i< 3; i ++){
        for(j = 0; j < 3; j ++){
            printf("Insira um valor na linha %d, coluna %d: ", i, j);
            scanf(" %d", &matriz[i][j]);
        }
    }

    for(i = 0; i< 3; i ++){
            soma += matriz[i][i];
    }
    printf("%d", soma);
    
    return 0;
}

File: 1_Período/Lista 13/Python/L13_Q01.py
matriz = []


for i in range(3):
    linha = []
    for j in range(3):
        linha.append(int(input(f'A{i}x{j} = ')))
    matriz.append(linha)

for linha in matriz:
    print(linha)

soma = 0
for i in range(3):
    soma += matriz[i][i]
    

print(soma)